console/st/console-st.sh

```diff
@@ -48,7 +48,7 @@ NODE_UUID=02e94b88-19ab-4166-b26b-91b51f22ad91  # required by `common.sh'
 start_server()
 {
 	modprobe lnet
-	modload_galois &>/dev/null
+	modload_galois
```

Comment:
Why?

Reply:
When "galois" module is still loaded (whether in-use or not), the modload_galois() function fails (and due to its stdout and stderr is re-directed to /dev/null), no error messages are printed to console. User will just see the script exit without any messages.

Remove the redirect will print our error messages before exit with error.